src/main/java/org/jabref/logic/importer/util/MediaTypes.java

```diff
@@ -5,4 +5,5 @@
  */
 public class MediaTypes {
     public static final String APPLICATION_BIBTEX = "application/x-bibtex";
+    public static final String APPLICATION_JSON = "application/vnd.citationstyles.csl+json";
```

Comment:
application/json is a different mime type. Could you rename the constant?

```suggestion
    public static final String CITATIONSTYLES_JSON = "application/vnd.citationstyles.csl+json";
```

Reply:
Done.